src/utils/parser.js

```diff
@@ -0,0 +1,68 @@
+import { types, transformFromAstSync } from '@babel/core'
+import { parseExpression } from '@babel/parser'
+import BrowserError from './error'
```

Comment:
Neat: Can we use named exports from `errors.js`?

Reply:
Will do